config/saucelabs.yml

```diff
@@ -0,0 +1,2 @@
+username: aballantine
+authkey: c09c4ae6-c68f-419d-b09a-922404c5bb23
```

Comment:
we can move saucelab's conf  in the `config/settings.yml`
e.g.
```
saucelabs:
  username: <%= ENV['SAUCELABS_USERNAME'] %>
  auth_key: <%= ENV['SAUCELABS_AUTH_KEY'] %>
```

then we can use env variable or the `.env` files to set the credentials